File: python/60 NumberGuessingGame.py
num=5
while True:
    guess=int(input("Enter a number (0 to 9): "))
    if guess>9 or guess<0:
        print("Out of range")
    elif guess>num:
        print("Guess lower")
    elif guess<num:
        print("Guess higher")
    elif guess==num:
        break
print("Game end")

File: java/60 NumberGuessingGame.java
package code;
import java.util.*;
public class Test {

	public static void main(String[] args) {
		Scanner sc=new Scanner(System.in);
		int num = 5;
		int guess;
		do {
			System.out.println("Enter a number (0 to 9): ");
			guess=sc.nextInt();
			if(guess>9 || guess<0) {
				System.out.println("Out of range");
			}
			else if(guess>num) {
				System.out.println("Guess lower");
			}
			else if(guess<num){
				System.out.println("Guess higher");
			}
		}while(num!=guess);
		System.out.println("Game end");
	}
}
